fms/modules/attention.py

```diff
@@ -704,10 +834,11 @@ def forward(
                 self.kvheads,
                 self.p_dropout if self.training else 0.0,
                 self.scale_factor,
-                **attn_kwargs,
+                **updated_attn_kwargs,
             )
 
         attn = attn.view(batch_size, q_len, self.nheads * self.emb_v_per_head)
+
         out = self.dense(attn)
 
         # if use_cache=True, we return the hidden_state as well as the kv cache
```

Comment:
Q: do we also need TP attention implementations? E.g. `has_sinks` is not a supported arg for `TPMultiHeadAttention` here. CC @ani300 ?